Sources/BraintreeVisaCheckout/BTVisaCheckoutNonce.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+#if canImport(BraintreeCard)
+import BraintreeCard
+#endif
+
+@objc public class BTVisaCheckoutNonce: BTPaymentMethodNonce {
+
+    @objc public let lastTwo: String
+    @objc public let cardType: String
+    @objc public let billingAddress: BTVisaCheckoutAddress
+    @objc public let shippingAddress: BTVisaCheckoutAddress
+    @objc public let userData: BTVisaCheckoutUserData
+    @objc public let callId: String
```

Comment:
Lets add docstrings to all of these: https://github.com/braintree/braintree_android/blob/main/VisaCheckout/src/main/java/com/braintreepayments/api/visacheckout/VisaCheckoutNonce.kt